hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/keyvalue/KvMetadataAvroResolver.java

```diff
@@ -43,6 +45,7 @@
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_KEY_AVRO_SCHEMA;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_VALUE_AVRO_RECORD_NAME;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_VALUE_AVRO_SCHEMA;
+import static com.hazelcast.jet.sql.impl.connector.file.AvroResolver.AVRO_TO_SQL;
```

Comment:
if that is going to be used in all avro mappings (file, kafka etc), we should move `AvroResolver` to `keyvalue` package

Reply:
we currently do not treat avro as full-fledged connector

Reply:
I am moving `AVRO_TO_SQL` to `KvMetadataAvroResolver` as a regular `Map`, and using `Map.getOrDefault(K, V)` in `AvroResolver`.

Reply:
do you mean that types explicitly mentioned in `AVRO_TO_SQL` are supported both for Kafka and File, but types not included there (ENUM, ARRAY, MAP, FIXED, STRING, BYTES) are supported only for File (file sql connector)?